lib/extract_messages.dart

```diff
@@ -237,7 +236,7 @@ class MessageFindingVisitor extends GeneralizingAstVisitor {
   /// encountered before seeing the Intl.message call.
   @override
   void visitMethodDeclaration(MethodDeclaration node) {
-    name = node.name.name;
+    name = node.name2.lexeme;
```

Comment:
For this (addressing an analyzer deprecation), do we need to increase the min. analyzer version in order to make sure name2 is available? Or, is this already captured in the analyzer dep in our pubspec?